index.js

```diff
@@ -1 +1,3 @@
-const eslint = require('eslint')
+const eslint2 = require('eslint')
+const abc = require('abc')
+const abc2 = require('abc')
```

Comment:
no-unused-vars: 'abc2' is assigned a value but never used  <!-- Automatically posted by Lintly -->